modules/storage/src/main/scala/zio/gcp/storage/Storage.scala

```diff
@@ -1,13 +1,307 @@
 package zio.gcp.storage
 
+import java.net.URL
+import java.util.concurrent.TimeUnit
+
+import com.google.api.gax.paging.Page
+import com.google.cloud.storage.HmacKey.HmacKeyState
+import com.google.cloud.storage.Storage._
+import com.google.cloud.storage.{ Acl, Blob, BlobId, BlobInfo, BucketInfo, CopyWriter, StorageBatch, StorageOptions }
+import com.google.cloud.{ Policy, ReadChannel, WriteChannel }
 import zio._
 
-trait Storage {
-  val storage: Storage.Service[Any]
-}
+import scala.jdk.CollectionConverters._
 
 object Storage {
-  trait Service[R] {
-    def create(bucketInfo: BucketInfo): URIO[R, BucketInfo]
+  trait Service {
+    def batch(): Task[StorageBatch]
```

Comment:
No need for `()` as it's a `Task`, not the side-effecting one.

Reply:
fixed